src/shared/components/schoolCard/schoolCard.css

```diff
@@ -2,10 +2,10 @@
   display: flex;
   border-radius: 4px;
   align-items: center;
-  margin: 20px 10px;
+  margin: 25px 10px;
```

Comment:
Ok so it's Firefox that is really causing these issues.
![image](https://user-images.githubusercontent.com/8031689/29642654-aa965fc8-882f-11e7-9318-691a7bc7fea9.png)